RELEASE/scripts/autoscend/iotms/mr2023.ash

```diff
@@ -450,21 +485,34 @@ void auto_buyFrom2002MrStore()
 		return;
 	}
 	auto_log_debug("Have " + remainingCatalogCredits() + " credit(s) to buy from Mr. Store 2002. Let's spend them!");
-	// meat butler on day 1 of run
+	/*// meat butler on day 1 of run
 	item itemConsidering = $item[meat butler];
 	if(have_campground() && remainingCatalogCredits() > 0 && my_daycount() == 1 && !haveCampgroundMaid() && auto_is_valid(itemConsidering))
 	{
 		buy($coinmaster[Mr. Store 2002], 1, itemConsidering);
 		use(itemConsidering);
-	}
+	}*/
 	// manual of secret door detection. skill: Secret door awareness
-	itemConsidering = $item[manual of secret door detection];
+	item itemConsidering = $item[manual of secret door detection];
 	if(can_read_skillbook(itemConsidering) && remainingCatalogCredits() > 0 && !auto_have_skill($skill[Secret door awareness]) && auto_is_valid(itemConsidering))
 	{
 		buy($coinmaster[Mr. Store 2002], 1, itemConsidering);
 		use(itemConsidering);
 	}
-	// giant black monlith. Mostly useful at low level for stats
+	//Pro skateboard to dupe tomb rat king drops
+	itemConsidering = $item[pro skateboard];
+	if(remainingCatalogCredits() > 0 && auto_is_valid(itemConsidering) && !possessEquipment(itemConsidering))
+	{
+		buy($coinmaster[Mr. Store 2002], 1, itemConsidering);
+	}
+	// meat butler on day 1 of run
+	itemConsidering = $item[meat butler];
+	if(have_campground() && remainingCatalogCredits() > 0 && my_daycount() == 1 && !haveCampgroundMaid() && auto_is_valid(itemConsidering))
+	{
+		buy($coinmaster[Mr. Store 2002], 1, itemConsidering);
+		use(itemConsidering);
+	}
+	/*// giant black monlith. Mostly useful at low level for stats
```

Comment:
I agree. I think I'm going to work on (in a separate PR) a levelling items categorizer where if we have a certain "score" we don't take an item in favor of utilizing other items because at some point we will have too much and it will be wasteful of resources like the Mr Store and the Censer